### PR TITLE
doc: add minutes for meeting 26 June 2024

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: meetings/2024-06-26.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+## Links
+
+* **Recording**:  <https://www.youtube.com/watch?v=DlrCDjGPKgQ>
+* **GitHub Issue**: <https://github.com/nodejs/TSC/issues/1585>
+
+## Present
+
+X* Antoine du Hamel @aduh95 (voting member)
+
+a *Benjamin Gruenbaum @benjamingr (voting member)
```
Comment (benjamingr) [inline meetings/2024-06-26.md:12]:
```suggestion
* Benjamin Gruenbaum @benjamingr (voting member)
```

File: meetings/2024-06-26.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+* No update this week, but board meeting is this Friday.
+
+### nodejs/node
+
+* doc: deprecate NaN or negative number as delay to setTimeout and setInterval [#53005](https://github.com/nodejs/node/pull/53005)
+  * Benjamin - believe we have achieved consensus. Going to change process to allow warnings
+    Outside of deprecations. Wanted warning if something that gets accidentally converted
+    To 0.
```
Comment (jakecastelli) [inline meetings/2024-06-26.md:48]:
```suggestion
    To 1.
```

Comment (benjamingr) [reply meetings/2024-06-26.md:48]: I said 0 in the meeting but this is technically correct

**Review by MoLow — APPROVED** [review]

**Review by richardlau — APPROVED** [review]

**Review by gireeshpunathil — APPROVED** [review]

File: meetings/2024-06-26.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+  * Benjamin - believe we have achieved consensus. Going to change process to allow warnings
+    Outside of deprecations. Wanted warning if something that gets accidentally converted
+    To 1.
+  * Process changing PR - <https://github.com/nodejs/node/pull/53513>
+  * Antoine, we don’t want to deprecate, that is because HTML spec does not want to deprecate
+
+### nodejs/TSC
+
+* Open OpenCollective and Github Sponsors for Node.js [#1553](https://github.com/nodejs/TSC/issues/1553)
+  * Still active but nothing specific to discussion. Agreed to remove agenda tag for noe.
```
Comment (UlisesGascon) [inline meetings/2024-06-26.md:55]:
```suggestion
  * Still active but nothing specific to discussion. Agreed to remove agenda tag for now.
```

**Pull request merged**